.github/ISSUE_TEMPLATE/07-feature_request.yml

```diff
@@ -0,0 +1,23 @@
+name: Feature Request
+description: requests for new ONNX Runtime features
+title: "[Feature Request] "
+labels: ["feature request"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before submitting your request, please review past submissions to ensure that it is not a duplicate of a known feature request.
+  - type: textarea
+    id: fr-descrip
+    attributes:
+      label: Describe the feature request
+      placeholder: A clear and concise description of the feature you would like, why it should be added, and any alternative solutions or features you've considered. Also provide any additional context necessary for your issue to be addressed (e.g., screenshots, GPU model & memory, etc.).
```

Comment:
The last sentence about in the placeholder does not seem necessary for feature requests.